vertical-pod-autoscaler/pkg/utils/vpa/capping_test.go

```diff
@@ -276,11 +281,218 @@ func TestApplyCapsToLimitRange(t *testing.T) {
 		},
 	}
 
-	calculator := fakeLimitRangeCalculator{limitRange}
+	calculator := fakeLimitRangeCalculator{containerLimitRange: limitRange}
 	processor := NewCappingRecommendationProcessor(&calculator)
 	processedRecommendation, annotations, err := processor.Apply(&recommendation, nil, nil, &pod)
 	assert.NoError(t, err)
 	assert.Contains(t, annotations, "container")
 	assert.ElementsMatch(t, []string{"cpu capped to fit Max in container LimitRange", "memory capped to fit Min in container LimitRange"}, annotations["container"])
 	assert.Equal(t, expectedRecommendation, *processedRecommendation)
 }
+
+func TestApplyPodLimitRange(t *testing.T) {
+	tests := []struct {
+		name         string
+		resources    []vpa_types.RecommendedContainerResources
+		pod          apiv1.Pod
+		limitRange   apiv1.LimitRangeItem
+		resourceName apiv1.ResourceName
+		expect       []vpa_types.RecommendedContainerResources
+	}{
+		{
+			name: "cap target cpu to max",
```

Comment:
Maybe also both min and max

Reply:
Test is now redundant. I'll change limit ranges to have both min and max.